pandas/core/generic.py

```diff
@@ -7832,12 +7832,13 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Notes
         -----
         The %(name)s method is an application of the if-then idiom. For each
-        element in the calling DataFrame, if ``cond`` is ``%(cond)s`` the
-        element is used; otherwise the corresponding element from the DataFrame
+        element in the caller, if ``cond`` is ``%(cond)s`` the
+        element is used; otherwise the corresponding element from
         ``other`` is used.
 
-        The signature for :func:`DataFrame.where` differs from
-        :func:`numpy.where`. Roughly ``df1.where(m, df2)`` is equivalent to
+        The signature for :func:`DataFrame.where` or
```

Comment:
Think you meant to say `Series` for one of these

Reply:
The problem is that this document will be used both for Series and for DataFrame. Therefore that paragraph would be odd in at least one of the two cases. I'm really not sure how to solve this.

Reply:
Just say `Series` for one and `DataFrame` for the other. Right now you are saying `DataFrame` for both

Reply:
I'm not sure what you mean there... There is one piece of documentation for both, how can I differentiate?

Reply:
My point is you don't need to differentiate (I mean you could, but its a lot more work and I don't think its worth the effort).

Right now you are saying "The signature for `DataFrame.where` or `DataFrame.where` differs from `numpy.where`". Just change that to "The signature for `Series.where` or `DataFrame.where` ..."

Reply:
I see! Sorry, it's just that I was hoping for a more automated solution. :-D

Reply:
There's probably a way to add substitution for say a `klass` keyword argument (we do this in other instances). Welcome to give it a shot if you like!

Reply:
I'll see what I can do!